Demo/Demo/Features/CardDemoViewController.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+class CardDemoViewController: FeatureBaseViewController {
+    // TODO: Validate card fields are filled out
+    // TODO: Add proper spacing / styling / max characters to card fields
+    // TODO: Animate pay button to indicate loading
+    // TODO: Disable pay button until fields are filled out
+
+    private enum Constants {
+        static let stackViewSpacing: CGFloat = 16
+        static let layoutSpacing: CGFloat = 16
+        static let textFieldHeight: CGFloat = 40
+    }
+
+    let mainStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .vertical
+        view.alignment = .center
+        view.spacing = Constants.stackViewSpacing
+        return view
+    }()
+
+    let secondaryStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .horizontal
+        view.alignment = .center
+        view.distribution = .fillEqually
+        view.spacing = Constants.stackViewSpacing
+        return view
+    }()
+
+    lazy var cardNumberTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "Card Number")
+        textField.keyboardType = .numberPad
+        return textField
+    }()
+
+    lazy var expirationTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "Expiration")
+        textField.keyboardType = .numberPad
+        return textField
+    }()
+
+    lazy var cvvTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "CVV")
+        textField.keyboardType = .numberPad
+        return textField
+    }()
+
+    lazy var payButton: UIButton = {
+        let payButton = CustomButton(title: "\(DemoSettings.intent.rawValue.capitalized) Order")
+        payButton.addTarget(self, action: #selector(didTapPayButton), for: .touchUpInside)
+        payButton.layer.cornerRadius = 8
+        payButton.backgroundColor = .systemBlue
+        payButton.tintColor = .white
+        return payButton
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .systemBackground
+
+        setupUI()
```

Comment:
very nit: I think we can either change `setupUI()` -> `addSubviews()`, or keep `setupUI()` but move `backgroundColor` in there. What do you think?